src/OSPSuite.Infrastructure.Import/Core/Mappers/DataSetToDataRepositoryMapper.cs

```diff
@@ -68,7 +69,7 @@ public DataSetToDataRepositoryMappingResult ConvertImportDataSet(ImportedDataSet
       private bool convertParsedDataColumnAndReturnWarningFlag(DataRepository dataRepository, KeyValuePair<ExtendedColumn, IList<SimulationPoint>> columnAndData, string fileName)
       {
          DataColumn dataColumn;
-         var unit = columnAndData.Value.FirstOrDefault(x => !string.IsNullOrEmpty(x.Unit)).Unit;
+         var unit = columnAndData.Value.FirstOrDefault(x => !string.IsNullOrEmpty(x.Unit))?.Unit ?? Constants.Dimension.DIMENSIONLESS;
```

Comment:
Dimensionless is a dimension not a unit
I meant to use NO_DIMENSION to match the code that you had before
NO_DIMENSION.DefaultUnit or sthg kike this